xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-test/xwiki-platform-image-lightbox-test-pageobjects/src/main/java/org/xwiki/image/lightbox/test/po/LightboxPage.java

```diff
@@ -41,6 +41,15 @@
  */
 public class LightboxPage extends ViewPage
 {
+    /**
+     * Make sure that the javascript is also loaded after page refresh since lightbox actions depend on it.
+     */
+    public void reloadPage()
+    {
+        getDriver().navigate().refresh();
+        waitUntilPageJSIsLoaded();
```

Comment:
This method seems generic so why not put it in some base class (like ``BasePage``)?